src/app/pages/onboarding/sign-in/utils/mnemonic-validation.ts

```diff
@@ -0,0 +1,24 @@
+import { getDefaultWordlist, wordlists } from 'bip39';
+import * as yup from 'yup';
+
```

Comment:
`validateMnemonic` only works for full keys: 'some key ...' so we need to get the users localised wordlist to validate against